### PR TITLE
reuse Equal

### PR DESCRIPTION
**Pull request by m3ngyang**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
clean up duplicated codes.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Comment by m3ngyang** [comment]
/assign @deads2k 
PTAL~

File: pkg/quota/v1/resources.go [hunk 1]
```diff
@@ -47,21 +47,7 @@ func Equals(a corev1.ResourceList, b corev1.ResourceList) bool {
 
 // V1Equals returns true if the two lists are equivalent
 func V1Equals(a corev1.ResourceList, b corev1.ResourceList) bool {
```
Comment (RainbowMango) [inline pkg/quota/v1/resources.go]: Should we eliminate redundant functions here?
 `func Equals(a corev1.ResourceList, b corev1.ResourceList) bool`  and `func V1Equals(a corev1.ResourceList, b corev1.ResourceList) bool`.

Bye the way, should we use `reflect.DeepEqual()` as the function implementation?

**Review by RainbowMango — COMMENTED** [review]
@yue9944882 
Maybe you can help.

File: pkg/quota/v1/resources.go (hunk 1 above)
Comment (jpbetz) [reply pkg/quota/v1/resources.go]: Eliminating `V1Equals` might like backward incompatible change (controllers can depend on this package, right?). But changing in-tree callers of `V1Equals` (I believe there is only one) to just call `Equals` instead might be reasonable.

Comment (m3ngyang) [reply pkg/quota/v1/resources.go]: Besides, the performance of `reflect.DeepEqual()` is much worse than the current implementation.
/assign @liggitt would you mind having a look.

Comment (liggitt) [reply pkg/quota/v1/resources.go]: go ahead and drop the V1Equals function, and update the single caller to call Equals

I agree we should not switch to using DeepEqual internally

Comment (m3ngyang) [reply pkg/quota/v1/resources.go]: `V1Equals` has been removed in the latest pull request. @liggitt

**Pull request merged**